webapp/config/packages/framework.yaml

```diff
@@ -4,6 +4,8 @@ framework:
     esi: false
     fragments: false
     http_method_override: true
+    #csrf_protection: true
+    annotations: true
```

Comment:
Isn't that the default already (https://symfony.com/doc/current/reference/configuration/framework.html#annotations)

Reply:
Yes but it will change so I’d rather be explicit